core/src/banking_stage.rs

```diff
@@ -254,7 +254,18 @@ impl BankingStage {
                 .unwrap_or_else(|_| buffered_packets.clear());
             }
 
-            match Self::process_packets(&verified_receiver, &poh_recorder, recv_start) {
+            let recv_timeout = if !buffered_packets.is_empty() {
```

Comment:
wish the sense of this were inverted, is a little harder to read !is_empty()

Reply:
@rob-solana  would ```buffered_packets.len() > 0``` be better?

Reply:
anything but !is_empty() ;)

Reply:
@rob-solana It seems clippy prefers !is_empty.

https://buildkite.com/solana-labs/solana/builds/5235#996a7087-8231-40dc-b408-f8f24bd0a961
```

if buffered_packets.len() > 0 {
--
  | \|                ^^^^^^^^^^^^^^^^^^^^^^^^^^ help: using `is_empty` is clearer and more explicit: `!buffered_packets.is_empty()`

```

Reply:
how about swapping the clauses?  if is_empty() default else special ?